Control coverage/Feature/APIManagement.md

```diff
@@ -222,7 +222,7 @@ Delegation allows you to use your existing website for handling developer sign-i
 
 	 To disable delegation, go to APIM service --> Developer portal --> Delegation --> Uncheck the two check boxes (a) Delegate sign-in & sign-up (b) Delegate product subscription. Read more about delegation here: https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-setup-delegation
 
-### Azure Policy or ARM API used for evaluation 
+### Azure Policies or REST APIs used for evaluation
 
 - ARM API to list APIMs and its related property at Subscription level: /subscriptions/{subscriptionId}/providers/Microsoft.ApiManagement/service?api-version=2019-12-01 <br />
```

Comment:
here we are not listing APIm but APIm services

Reply:
Done, changed it to REST API to list APIMs services